pkg/monitor/cluster/healthz.go

```diff
@@ -21,10 +21,10 @@ func (mon *Monitor) emitAPIServerHealthzCode(ctx context.Context) error {
 		"code": strconv.FormatInt(int64(statusCode), 10),
 	})
 
-	return err
+	return statusCode, err
 }
 
-func (mon *Monitor) getAPIServerPingCode(ctx context.Context) (int, error) {
```

Comment:
Could we still return the status code? We'll use this in the metric.

Reply:
The status code is emitted within the `emitAPIServerPingCode` function